python/src/gz/sim/_gz_sim_pybind11.cc

```diff
@@ -28,7 +30,7 @@
 
 PYBIND11_MODULE(BINDINGS_MODULE_NAME, m) {
   m.doc() = "Gazebo Sim Python Library.";
-
+  m.attr("k_null_entity") = gz::sim::kNullEntity;
```

Comment:
super nit: 
```suggestion
  m.attr("K_NULL_ENTITY") = gz::sim::kNullEntity;
```

or just `NULL_ENTITY`.

Reply:
Done.